tests/test_external_resource_error.py

```
@@ -0,0 +1,287 @@
+        assert status_code == "403"
+        assert "upload.wikimedia.org" in url
+
+    def test_vllm_wrapped_error_without_cause(self):
+        """Test vLLM wrapped errors without the original cause details."""
+        # This is the case where only the 500 wrapper is visible
+        error = Exception(
+            "openai.InternalServerError: Error code: 500 - {'error': {'message': "
+            "'litellm.InternalServerError: InternalServerError: OpenAIException - "
+            "Internal Server Error. Received Model Group=Qwen2.5-VL-7B-Instruct'}}"
+        )
+        is_external, status_code, url = _is_external_resource_error(error)
```

Ruff (0.14.10)

Unpacked variable `is_external` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)